dev/com.ibm.ws.security.kerberos.auth/src/com/ibm/ws/security/kerberos/auth/Krb5LoginModuleWrapper.java

```diff
@@ -46,8 +45,7 @@ private static boolean isIBMLoginModuleAvailable() {
     }
 
     // Cannot rely purely on JavaInfo.vendor() because IBM JDK 8 for Mac OS reports vendor = Oracle and only has some IBM API available
-    private static final boolean isIBMJdk8 = JavaInfo.majorVersion() <= 8 &&
-                                             (JavaInfo.vendor() == Vendor.IBM || isIBMLoginModuleAvailable());
+    private static final boolean isIBMJdk8 = isIBMLoginModuleAvailable();
```

Comment:
Probably a good idea to change this variable name.

Reply:
`isIBMLoginModuleAvailable` should probably be replaced with `JavaInfo.isAvailable` as well ... but since the aim of this PR is simply not to use vendor, the change can be made under a separate PR.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/18054